### DISCUSSION TITLE
Keybindings not working when a Cyrillic keyboard layout is chosen

### DISCUSSION THREAD

**Discussion by s-aleck** [qtile/qtile#3255] [Q&A]
I have an issue with Qtile on Arco (in a virtual machine). When I activate a Cyrillic keyboard layout, the keybindings which use letter keys stop working. I tried setting up parallel keybindings which use Cyrillic letters, but it didn’t work.

How can I use all of my keybindings regardless of the active layout? Thanks.

EDIT I was hoping there was a way to refer to keys in a way which is above layouts, such as numerical codes.

**Comment by elParaguayo** [comment]
This has been discussed a bit before (have a look at #933) but nothing's been implemented yet.
